### DISCUSSION TITLE
Support Vue3

### DISCUSSION THREAD

**Discussion by AlanLes** [Hexastack/eazychart#38] [Ideas]
Hello guys, I noticed that the library has support for vue2. It happens that I feel pretty confident with the latest Vue3. Are you going to support it? I could help with migration; I'd love to. 🙂

**Comment by marrouchi** [comment]
Hello @AlanLes, This would be amazing to have ! If you are in, we could prioritize this and work on it together. Want to take the lead on this ?

I wonder if : 
1. We should create another package, using "Vite" ...
2. Or, update the current one so it could be retro-compatible with Vue2

Let me know what you think :)

**Reply by marrouchi** [reply]
@AlanLes FYI I'm working on a big PR here https://github.com/Hexastack/eazychart/pull/32, in which there's update about how we use scales.

Just so you know before to start working on Vue3, maybe better wait for this one. I should be able to finish it in the next couple of days.

**Reply by AlanLes** [reply]
Yep, I think it may be better to wait for this to be completed! 🤞

**Comment by AlanLes** [comment]
@marrouchi yep, sure, it would be great to work on that; we should prioritize this and work on it together! I'm not sure if we should mess around with the existing package, especially if it does work with vue2 so far. It may be nice having both packages, or at least for some time.

**Reply by marrouchi** [reply]
Yes, you are right. Wise choice :+1:

**Comment by marrouchi** [comment]
Hello @AlanLes I forgot to tell you that you could start on the vue3 implementation now :rocket:

**Reply by AlanLes** [reply]
Nice! 🚀  Are we going to write some tasks/issues for that? Where should I start, what's your suggestion? 😄 
Ah, and I'm not sure how will I be on spare time, because next week probably my son will be born 🙈 👶 👨‍🍼

**Reply by marrouchi** [reply]
Oh congratulations :partying_face: ! Family comes first, I will see how we could organize the work around this and keep you posted ;)

**Reply by AlanLes** [reply]
Great news! Yes, family comes first, so I'll indeed have some vacation time during the first two weeks. Anyway, I'm 100% willing to help you with that and implement these Vue3 parts!
